admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -372,3 +393,7 @@ You must run ``occ`` as your HTTP user; see
 
 When you are finished, take your ownCloud server out of 
 ``maintenance:singleuser`` mode.
+
+.. Links
+   
+.. _User Home Folder Naming Rule: https://doc.owncloud.com/server/9.1/admin_manual/configuration_user/user_auth_ldap.html#special-attributes
```

Comment:
> Gladly, it's not required.

If you mean that the version-specific part is not needed then I agree. This setting should be present already in the current version (and any past versions)